src/baldr/edgeinfo.cc

```diff
@@ -200,12 +227,11 @@ std::vector<std::string> EdgeInfo::GetTaggedValues(bool only_pronunciations) con
     if (ni->name_offset_ < names_list_length_) {
       const char* value = names_list_ + ni->name_offset_;
       try {
-        // due to short sightedness on pronunciations we have a dichotomy here
-        // basically you can either get other stuff OR pronunciations not both..
```

Comment:
something like this comment still stands,  you are explicitly skipping tagged values which are kLinguistic, the only difference now is you dont have the choice. i feel like this is a gotcha, is the javadoc for the function updated to say kLinguistic is skipped?

Reply:
updated the comments